app/scripts/pages/logs.js

```diff
@@ -11,7 +10,7 @@ var Logs = React.createClass({
     host: React.PropTypes.string
   },
   getInitialState: function () {
-    var t = this
+    // var t = this
     var req = null
     // TODO: fix this
     // var req = this.props.ipfs.log.tail(function (err, stream) {
```

Comment:
tail now works through js-ipfs-api! one more thing to add to the webui as a feature //cc @RichardLitt :)